spec/features/create_image_spec.rb

```diff
@@ -0,0 +1,33 @@
+# Generated via
+#  `rails generate hyrax:work Image`
+require 'rails_helper'
+include Warden::Test::Helpers
+
+# NOTE: If you generated more than one work, you have to set "js: true"
+RSpec.feature 'Create a Image', js: false do
+  context 'a logged in user' do
+    let(:user_attributes) do
+      { email: 'test@example.com' }
+    end
+    let(:user) do
+      User.new(user_attributes) { |u| u.save(validate: false) }
+    end
+
+    before do
+      AdminSet.find_or_create_default_admin_set_id
+      login_as user
+    end
+
+    scenario do
+      visit '/dashboard'
+      click_link "Works"
+      click_link "Add new work"
+
+      # If you generate more than one work uncomment these lines
```

Comment:
This test doesn't seem to be running in CI, we'll want to look into that before moving forward with the other types, too.